ini_test.go

```diff
@@ -87,208 +87,982 @@ NAME = Unknwon
+url   = https://github.com/Antergone/test1.git
+url   = https://github.com/Antergone/test2.git
+fetch = +refs/heads/*:refs/remotes/origin/*
+
+`)
+				})
+
+				Convey("Inverse case", func() {
+					f, err := ini.LoadSources(ini.LoadOptions{AllowPythonMultilineValues: false}, []byte(`
+[remote "origin"]
+url = https://github.com/Antergone/test1.git
+url = https://github.com/Antergone/test2.git`))
+					So(err, ShouldBeNil)
+					So(f, ShouldNotBeNil)
+
+					So(f.Section(`remote "origin"`).Key("url").String(), ShouldEqual, "https://github.com/Antergone/test2.git")
+				})
+			})
+
+			Convey("Unescape double quotes inside value", func() {
+				f, err := ini.LoadSources(ini.LoadOptions{
+					AllowPythonMultilineValues: false,
+					UnescapeValueDoubleQuotes:  true,
+				}, []byte(`
 create_repo="创建了仓库 <a href=\"%s\">%s</a>"`))
 				So(err, ShouldBeNil)
 				So(f, ShouldNotBeNil)
 
-				So(f.Section("").Key("create_repo").String(), ShouldEqual, `"创建了仓库 <a href=\"%s\">%s</a>"`)
+				So(f.Section("").Key("create_repo").String(), ShouldEqual, `创建了仓库 <a href="%s">%s</a>`)
+
+				Convey("Inverse case", func() {
+					f, err := ini.LoadSources(ini.LoadOptions{AllowPythonMultilineValues: false}, []byte(`
+create_repo="创建了仓库 <a href=\"%s\">%s</a>"`))
+					So(err, ShouldBeNil)
+					So(f, ShouldNotBeNil)
+
+					So(f.Section("").Key("create_repo").String(), ShouldEqual, `"创建了仓库 <a href=\"%s\">%s</a>"`)
+				})
 			})
-		})
 
-		Convey("Unescape comment symbols inside value", func() {
-			f, err := ini.LoadSources(ini.LoadOptions{
-				IgnoreInlineComment:         true,
-				UnescapeValueCommentSymbols: true,
-			}, []byte(`
+			Convey("Unescape comment symbols inside value", func() {
+				f, err := ini.LoadSources(ini.LoadOptions{
+					AllowPythonMultilineValues:  false,
+					IgnoreInlineComment:         true,
+					UnescapeValueCommentSymbols: true,
+				}, []byte(`
 key = test value <span style="color: %s\; background: %s">more text</span>
 `))
-			So(err, ShouldBeNil)
-			So(f, ShouldNotBeNil)
+				So(err, ShouldBeNil)
+				So(f, ShouldNotBeNil)
 
-			So(f.Section("").Key("key").String(), ShouldEqual, `test value <span style="color: %s; background: %s">more text</span>`)
-		})
+				So(f.Section("").Key("key").String(), ShouldEqual, `test value <span style="color: %s; background: %s">more text</span>`)
+			})
+
+			Convey("Cant't parse small python-compatible INI files", func() {
+				f, err := ini.LoadSources(ini.LoadOptions{AllowPythonMultilineValues: false}, []byte(`
+[long]
+long_rsa_private_key = -----BEGIN RSA PRIVATE KEY-----
+   foo
+   bar
+   foobar
+   barfoo
+   -----END RSA PRIVATE KEY-----
+`))
+				So(err, ShouldNotBeNil)
+				So(f, ShouldBeNil)
+				So(err.Error(), ShouldEqual, "key-value delimiter not found: foo\n")
+			})
+
+			Convey("Cant't parse big python-compatible INI files", func() {
+				f, err := ini.LoadSources(ini.LoadOptions{AllowPythonMultilineValues: false}, []byte(`
```

Comment:
What is difference between this one and [line 106](https://github.com/go-ini/ini/pull/145/files?diff=split#diff-13c7fb9c9c1b5a36d0c1d37eab6dbf9dR106)?

Reply:
I copy-pasted badly. :clown_face:  It must be `ini.Load`